### ISSUE TITLE
Is this repository active?

### ISSUE THREAD

**Issue by ramtob** [DanielYKPan/date-time-picker#675]
Or was it abandoned? Anybody knows?

**Comment by johannesjo** [comment]
Looks pretty abandoned to me :(

**Comment by danielmoncada** [comment]
Yeah, might be abandoned. Though, the author did merge a PR in September of last year.

Might need to make a new fork =\

**Comment by johannesjo** [comment]
@danielmoncada there already are several. I wish GitHub would offer a better way to deal with these scenarios..

**Comment by danielmoncada** [comment]
@johannesjo yeah I agree. I need this updated to support ng9. Might just fork my own.

**Comment by jcompagner** [comment]
If @DanielYKPan does not maintain this anymore, can we somehow make 1 fork with a group of people (so not 1 comitter) that then maintains this project? (and makes releases)

**Comment by johannesjo** [comment]
@jcompagner good idea! Personally I am quite busy with my own open source projects, so I probably won't be able to spent too much time on this, but I can of course create a fork and add anyone as a collaborator who is interested.

**Comment by danielmoncada** [comment]
My team forked this already. I've updated it to ng9. There are a few things that still need to be fixed in case anyone wants to work on it (before we do).

https://github.com/danielmoncada/date-time-picker

Pull it, and run 

```cs
ng build picker
```
Looks like there are stricter linting rules that cause the build failure

**Comment by danielmoncada** [comment]
Update: current master on https://github.com/danielmoncada/date-time-picker is up to date, and works with Ivy and Ng9. Will publish an npm package in the morning.

**Comment by jcompagner** [comment]
Nice, where you also able to look at the moment package? Because I was always still at 1.0.7 because for some reason 1.0.8 never worked for me.

**Comment by danielmoncada** [comment]
@jcompagner I have it currently using 2.24

**Comment by jcompagner** [comment]
yeah i also use that but i was talking about the:

    "moment": "^2.24.0",
    "ng-pick-datetime": "^7.0.0",
    "ng-pick-datetime-moment": "<=1.0.7", <<< this package

i think that is needed for using moment right?

**Comment by danielmoncada** [comment]
@jcompagner it works, but I do get some peer dependency warnings. There's really no way to fix them, as it appears that his repo for 'ng-pick-datetime-moment' is now gone

Edit: I found a fork for it: https://github.com/davidbonachera/ng-pick-datetime-moment

**Comment by jcompagner** [comment]
but isnt that just:
https://github.com/DanielYKPan/date-time-picker/tree/master/projects/picker/src/lib/date-time/adapter/moment-adapter

(but then the actual typescript stuff)

did he really just move it in the package itself? so it is always just included?

**Comment by jcompagner** [comment]
yes that dir really looks just like what we already have in the package itself:

node_modules\ng-pick-datetime\date-time\adapter\moment-adapter

**Comment by danielmoncada** [comment]
@jcompagner  you're right! I was referring to the name you provided me in your comment above 'ng-pick-datetime-moment'. Though, since it's already part of the package itself now, looks like we can handle / manage it a lot better now.

**Comment by danielmoncada** [comment]
@jcompagner @johannesjo npm now published: https://www.npmjs.com/package/@danielmoncada/angular-datetime-picker

**Comment by jcompagner** [comment]
i am testing it now, i do get:

WARNING in Entry point '@danielmoncada/angular-datetime-picker' contains deep imports into 'C:/Users/jcomp/git/servoy_ngclient2/servoy-eclipse/com.servoy.eclipse.ngclient.ui/node/node_modules/moment/moment'. This is probably not a problem, but may cause the compilation of entry points to be out of order.

not sure what that is

I am currently testing the moment stuff because this line:

https://github.com/DanielYKPan/date-time-picker/blob/master/projects/picker/src/lib/date-time/adapter/moment-adapter/moment-date-time.module.ts#L26

i did before in my own code, but it seems to me i just need to depend on that module right?

**Comment by jcompagner** [comment]
i guess that warning is about:

https://github.com/DanielYKPan/date-time-picker/blob/master/projects/picker/src/lib/date-time/adapter/moment-adapter/moment-date-time-adapter.class.ts#L7

because in my code i just do:

import * as moment from 'moment';

**Comment by danielmoncada** [comment]
@jcompagner I'll look into the warning

**Comment by danielmoncada** [comment]
@jcompagner see latest package (9.1.1)

**Comment by rskendzic** [comment]
> @jcompagner @johannesjo npm now published: https://www.npmjs.com/package/@danielmoncada/angular-datetime-picker

Not all hero wear capes. Thank you, sir!

**Comment by jcompagner** [comment]
@danielmoncada yes all warnings are gone, and it seems to work nicely now.
